app/src/main/java/com/team/uptech/pomodoro/domain/interactor/ChangeSettingsUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.team.uptech.pomodoro.domain.interactor
+
+import com.team.uptech.pomodoro.data.model.Pomodoro
+import io.reactivex.Completable
+import io.reactivex.Single
+
+/**
+ * Created on 22.05.17.
+ */
+interface ChangeSettingsUseCase {
+    fun getPomodoroTypeTime(type: Pomodoro): Single<Int>
+    fun getIsInfinite(): Single<Boolean>
+    fun changeTypeTime(type: Pomodoro, time: Int): Completable
+    fun changeInfinity(isInfinite: Boolean): Completable
```

Comment:
changeInfinity() sounds rather dramatical, don't you think?)
I would name it changeIsInfinite() or changeIsRunningInfinite()